plugins/default/src/Ide/Plugin/Pragmas.hs

```diff
@@ -40,35 +43,32 @@ descriptor plId = (defaultPluginDescriptor plId)
 -- Pragma is added to the first line of the Uri.
 -- It is assumed that the pragma name is a valid pragma,
 -- thus, not validated.
-mkPragmaEdit :: Uri -> T.Text -> WorkspaceEdit
-mkPragmaEdit uri pragmaName = res where
-    pos = J.Position 0 0
-    textEdits = J.List
-      [J.TextEdit (J.Range pos pos)
-                  ("{-# LANGUAGE " <> pragmaName <> " #-}\n")
-      ]
-    res = J.WorkspaceEdit
-      (Just $ H.singleton uri textEdits)
-      Nothing
+mkPragmaEdit :: Uri -> T.Text -> Range -> Maybe WorkspaceEdit
+mkPragmaEdit uri pragmaName rng = res where
+    textEdits = J.List [J.TextEdit rng ("{-# LANGUAGE " <> pragmaName <> " #-}\n")]
+    res = Just $ J.WorkspaceEdit (Just . H.singleton uri $ textEdits) Nothing
```

Comment:
Do we need to move this `Just` into here? I mean, the `Just` at `Just $ J.WorkspaceEdit ...` (it was originally on line 69)